introduction.md

```diff
@@ -0,0 +1 @@
+Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ducimus incidunt, nisi numquam ad, cumque mollitia illo! Eius animi voluptates corporis sequi in quod ipsam! Explicabo.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.